packages/web-twig/tests/__snapshots__/ComponentsSnapshotTest__test with data set modalComposedDefault.twig__1.html

```diff
@@ -0,0 +1,52 @@
+<html><body>
```

Comment:
Oh, this snapshot is a little bit messy… 🙈 But the other one looks correct to me 🙂.